assets/js/base/utils/errors.js

```diff
@@ -1,9 +1,22 @@
-export const formatError = ( apiError ) => {
-	return typeof apiError === 'object' && apiError.hasOwnProperty( 'message' ) ? {
-		apiMessage: apiError.message,
-	} : {
-		// If we can't get any message from the API, set it to null and
-		// let <ApiErrorPlaceholder /> handle the message to display.
-		apiMessage: null,
-	};
+const formatErrorMessage = ( error, messageProperty = 'frontendMessage' ) => {
+	if ( typeof error === 'object' && error.hasOwnProperty( 'message' ) ) {
+		return {
+			[ messageProperty ]: error.message,
+		};
+	}
+
+	return error;
+};
+
+export const formatError = async ( error ) => {
```

Comment:
Should have some jsdoc blocks here describing the expected shape of `error`.  I'm assuming it's intended to be used with a response from something using the `fetch` interface?

Reply:
Added in 2ac40b02d47db9e640d3eee79b95aa4b115f482a.